FFXIVClientStructs/FFXIV/Client/UI/Misc/CharaView.cs

```diff
@@ -1,47 +1,98 @@
 ﻿using FFXIVClientStructs.FFXIV.Client.Game;
+using FFXIVClientStructs.FFXIV.Client.Game.Character;
+using FFXIVClientStructs.FFXIV.Client.System.Memory;
+using FFXIVClientStructs.FFXIV.Component.GUI;
 
 namespace FFXIVClientStructs.FFXIV.Client.UI.Misc;
 
 // Client::UI::Misc::CharaView
+//
+// ClientObjectIndex:
+//  0 used in Character, PvPCharacter
+//  1 used in Inspect, CharaCard
+//  2 used in TryOn, GearSetPreview
+//  3 used in Colorant
+//  4 used in BannerList, BannerEdit
+//  0 - 7 used in BannerParty
+//
 [StructLayout(LayoutKind.Explicit, Size = 0x2C8)]
-public unsafe partial struct CharaView
+public unsafe partial struct CharaView : ICreatable
 {
     [FieldOffset(0x0)] public void** VTable;
```

Comment:
this will throw an error because the generator creates that same field for the vfuncs

Reply:
😮 Should I remove VTable or ICreatable?

Reply:
Wait, which generator? I didn’t add any attributes. The VTable field already existed.